src/runtime/micro/micro_common.h

```diff
@@ -0,0 +1,279 @@
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file micro_common.h
+ */
+#ifndef TVM_RUNTIME_MICRO_MICRO_COMMON_H_
+#define TVM_RUNTIME_MICRO_MICRO_COMMON_H_
+
+#include <stdio.h>
+
+#include <tvm/runtime/registry.h>
+
+#include <sstream>
+#include <string>
+#include <unordered_map>
+
+namespace tvm {
+namespace runtime {
+/*!
+ * \brief enum of device memory region sections
+ */
+enum SectionKind : int {
+  kText = 0,
```

Comment:
Do we need to add prefix to these constants?

Reply:
The leading "k" seems to be the convention, unfortunately.

Reply:
I mean we may need an additional prefix between the leading `k` and `Text`, like the definition of `kDLFloat` in the dlpack. Without such prefix, a developer might confuse between `SectionKind` and other enums, because `kText` and `kString` seems to belong to a same enum.

Reply:
I don't have any strong feelings about adding a prefix like `Micro` to them, but I'm unsure of the conventions around enums.  CC @tqchen

Reply:
I just have a feeling that someone might confuse some of the constants in this enum with `kStr` and `kBytes` in `TVMTypeCode` enum.

Reply:
Let us use enum class, which will enforce the prefix